jdaviz/configs/cubeviz/plugins/slice/tests/test_slice.py

```diff
@@ -58,6 +58,13 @@ def test_slice(cubeviz_helper, spectrum1d_cube):
     cubeviz_helper.select_wavelength(4.62360028e-07)
     assert sl.value == slice_values[1]
 
+    # Add test for unit conversion
+    uc_plugin = cubeviz_helper.plugins['Unit Conversion']._obj
+    uc_plugin.spectral_unit.selected = 'Angstrom'
+    assert sl.value_unit == 'Angstrom'
+    cubeviz_helper.select_wavelength(4623.60028)
+    assert sl.value == 1
```

Comment:
the value should be the wavelength, not the slice number (so I'd expect `assert sl.value == 4623.60028`)

Reply:
Yes this commit is outdated.

Reply:
its showing in the full PR diff 🤷‍♂️